tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -629,17 +662,35 @@ platform_tests/test_reboot.py:
     conditions:
       - "hwsku in ['Celestica-E1031-T48S4']"
 
+platform_tests/test_reboot.py::test_fast_reboot:
+  skip:
+    reason: "Skip test_fast_reboot for Nokia-7215"
+    conditions:
+      - "platform in ['armhf-nokia_ixs7215_52x-r0']"
+
 platform_tests/test_reboot.py::test_power_off_reboot:
   skip:
-    reason: "Skip power off reboot test for Wistron"
+    reason: "Skip power off reboot test for Wistron/Nokia-7215"
+    conditions:
+      - "hwsku in ['Celestica-E1031-T48S4'] or 'sw_to3200k' in hwsku or platform in ['armhf-nokia_ixs7215_52x-r0']"
+
+platform_tests/test_reboot.py::test_soft_reboot:
+  skip:
+    reason: "Skip test_soft_reboot for Nokia-7215"
+    conditions:
+      - "platform in ['armhf-nokia_ixs7215_52x-r0']"
+
+platform_tests/test_reboot.py::test_warm_reboot:
+  skip:
+    reason: "Skip test_warm_reboot for Nokia-7215"
     conditions:
-      - "hwsku in ['Celestica-E1031-T48S4'] or 'sw_to3200k' in hwsku"
+      - "platform in ['armhf-nokia_ixs7215_52x-r0']"
 
 platform_tests/test_reboot.py::test_watchdog_reboot:
   skip:
-    reason: "Skip watchdog reboot test for Wistron"
+    reason: "Skip watchdog reboot test for Wistron/Nokia-7215"
     conditions:
-      - "hwsku in ['Celestica-E1031-T48S4'] or 'sw_to3200k' in hwsku"
+      - "hwsku in ['Celestica-E1031-T48S4'] or 'sw_to3200k' in hwsku or platform in ['armhf-nokia_ixs7215_52x-r0']"
```

Comment:
skip for m0, but we may need this enabled for m0 in future

Reply:
Fixed